thread.c

```diff
@@ -77,6 +77,11 @@
 #include "mjit.h"
 #include "hrtime.h"
 
+#ifdef __linux__
```

Comment:
is it needed?

Reply:
Yes. Otherwise, `alloca` takes slow path.